src/js/tabs/trade.controller.js

```diff
@@ -615,8 +615,13 @@ TradeTab.prototype.angular = function(module)
       tx.on('success', function(res) {
         setEngineStatus(res, true, type);
 
+
         if (!modifying) order.mode = 'done';
 
+        $timeout(function() {
```

Comment:
What is this timeout for? Does it reset the form after 2 seconds? What if I started typing another offer?